tests/cephadm/test_cluster_view.py

```diff
@@ -0,0 +1,114 @@
+from json import loads
+
+from ceph.waiter import WaitUntil
+from cli.cephadm.cephadm import CephAdm
+from utility.log import Log
+
+log = Log(__name__)
+
+DEFAULT_CEPH_DIR = "/etc/ceph/"
+DEFAULT_CONF_PATH = "/etc/ceph/ceph.conf"
+
+
+class ClusterViewError(Exception):
+    pass
+
+
+def verify_mon_service(host):
+    """
+    Verify MON service is up
+    Args:
+        host(ceph): Installer host
+    """
+    timeout, interval = 30, 5
+    for w in WaitUntil(timeout=timeout, interval=interval):
+        configs = loads(CephAdm(host).ceph.orch.ls(service_type="mon", format="json"))
+        for config in configs:
+            running = config.get("status", {}).get("running")
+            size = config.get("status", {}).get("size")
+        if running == size:
+            log.info("All MON services are up and running")
+            return True
+        if w.expired:
```

Comment:
@adityaramteke As per logic, after the `ceph orch apply mon --host "ceph-aramteke-automation-new-ywldzh-node1-installer ceph-aramteke-automation-new-ywldzh-node2 ceph-aramteke-automation-new-ywldzh-node3"` the running==size should be true and ` All MON services are up and running` should be mentioned in the log. But here I can see the waituntil has completed and the if statement was never satisfied. 

Also its strange to see that the exception was also not raised. The test should fail